client/src/App.test.tsx

```diff
@@ -19,11 +19,7 @@ jest.unmock('react-toastify')
 const apiMocks = {
   failedAuth: {
     url: '/api/me',
-    response: {},
-    error: {
-      status: 401,
-      statusText: 'UNAUTHORIZED',
-    },
+    response: null,
```

Comment:
I'm a little bit irked by a null response vs an empty object... but I think I can live with it.

Reply:
@benadida didn't you make this change? https://github.com/votingworks/arlo/pull/795

Reply:
Lol, I did. I did. I remember worrying about it and then saying screw it. And apparently I forgot about the "screw it" part.

Never mind.